runtime/doc/deprecated.txt

```diff
@@ -202,9 +213,6 @@ internally and are no longer exposed as part of the API. Instead, use
 - *vim.lsp.diagnostic.set_underline()*
 - *vim.lsp.diagnostic.set_virtual_text()*
 
-Configuring |diagnostic-signs| with |:sign-define| or |sign_define()| is no
-longer supported. Use the "signs" key of |vim.diagnostic.config()| instead.
-
```

Comment:
Is this supported now?

Reply:
This was duplicated, it's listed twice in `deprecated.txt` so I removed the (incorrect) one.